animations/spells/1st/vitality-lash.json

```diff
@@ -0,0 +1,29 @@
+{
+	"item:slug:vitality-lash": [
+		{
+			"trigger": "spell-cast",
```

Comment:
The spell is a damaging fortitude save spell. Use either `saving-throw` or `damage-taken`.

Reply:
i was thinking it was more a you strike out and they take fortitude with vitality

IE a `damage-roll` ranged

but i can redo it as damage taken or saving throw, but i'd need to change the animation to be on token instead of ranged

Reply:
saving-throw should work with ranged 🤔